packages/deposit_firestore/lib/src/firestore_deposit_adapter.dart

```diff
@@ -95,6 +104,15 @@ class FirestoreDepositAdapter extends DepositAdapter<String> {
     await doc?.delete();
   }
 
+  @override
+  Future<void> removeAll(
+    String table,
+    String primaryColumn,
+    List<Map<String, dynamic>> data,
+  ) async {
```

Comment:
```suggestion
  ) {
```

Reply:
This one has to stay async cause the Future.wait is not returned, but you do want to wait until it is finished